test/system/application_system_test_case.rb

```diff
@@ -9,4 +9,11 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
 
   Capybara.server = :puma, { Silent: true }
   Selenium::WebDriver.logger.ignore(:browser_options)
+
+  # TODO: Remove this hack
```

Comment:
@katjam - not the prettiest but i think if we don't get this passing we can't deploy. i've made a tech debt ticket.

Reply:
Seems like a reasonable hack though.